Documentation/gettingstarted/bandwidth-manager.rst

```diff
@@ -11,7 +11,7 @@ Bandwidth Manager (beta)
 ************************
 
 This guide explains how to configure Cilium's bandwidth manager to
-optimize TCP and UDP workloads and efficiently rate limit individual Pods
+optimize TCP and UDP workloads and efficiently rate limit individual pods
```

Comment:
I would prefer to have Pods with capital spelling. This is also consistent elsewhere, e.g. https://cloud.google.com/kubernetes-engine/docs/concepts/pod .